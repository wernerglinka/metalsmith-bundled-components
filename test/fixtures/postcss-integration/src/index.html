---
layout: base.njk
sections:
  - banner
  - footer
---