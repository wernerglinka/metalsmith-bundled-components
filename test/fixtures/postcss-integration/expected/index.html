<!DOCTYPE html>
<html>
<head>
  <title>PostCSS Integration Test</title>
  <link rel="stylesheet" href="/assets/main.css">
</head>
<body>
  <div class="banner">
    <h1 class="banner__title">PostCSS Integration Test</h1>
  </div>
  
  <script src="/assets/main.js"></script>
</body>
</html>